src/components/layout/CyVerseAppBar.js

```diff
@@ -231,8 +231,7 @@ function CyverseAppBar(props) {
         config: {
             enabled: true,
             onSuccess: setUserProfile,
-            cacheTime: Infinity,
-            staleTime: Infinity,
+            refetchInterval: clientConfig.sessions.poll_interval_ms,
```

Comment:
I ended up changing the `?.` operators to simply `.` because the value should be defined everywhere.

Reply:
@slr71 - removing the `?` throws an error on `Sonora` start-up. I guess `clientConfig` prop could be `null` during startup ? 🤔

Reply:
You could run the profile query after `config` state is set.

Reply:
I didn't notice that error. Thanks for catching that. I'll update it today.